README.md

```diff
@@ -1,23 +1,23 @@
-# legion-airflow-plugins
+# odahu-airflow-plugins
 
-An apache airflow plugin for the Legion Platform.
+An apache airflow plugin for the Odahu Platform.
 
 ### How-To 
 
-1. Start a Legion Cluster and authenticate using OAuth2. (Hit the cluster's base URL.) Copy the token to line 8 of ./dags/legion_dag.py
+1. Start a Odahu Cluster and authenticate using OAuth2. (Hit the cluster's base URL.) Copy the token to line 8 of ./dags/odahu_dag.py
 1. [Install, Start and InitDB for Airflow](https://airflow.apache.org/installation.html)  
 1. Install the plugin defined in ./plugin/ to Airflow (or softlink/copy to $AIRFLOW_HOME/plugins)
-1. Copy or import ./dags/legion_dag.py
+1. Copy or import ./dags/odahu_dag.py
 1. Create the Airflow connections:
-* `legion_model`
-    * `id`: legion_model
+* `odahu_model`
+    * `id`: odahu_model
     * `type`: http
     * `host`: edge.cluster-name.ailifecycle.org
     * `schema`: https
-* `legion_edi`
-    * `id`: legion_edi
+* `odahu_api`
+    * `id`: odahu_edi
     * `type`: http
-    * `host`: edi.cluster-name.ailifecycle.org
+    * `host`: odahu.cluster-name.ailifecycle.org
```

Comment:
I don’t think it’s a good idea to specify a real domain here. It might be worth using something like `example.com` or something.